### ISSUE TITLE
Is it possable to render stills images with this addon?

### ISSUE THREAD

**Issue by gumanoed** [WARP-LAB/Blender-Network-Render-Additions#2]
Hallo!
I'm looking for an easy way to render big still image (not sequence for animation) with power of 12 PC which we have on classrooms here in college.

I read info on the readme of this project and i do not get it clear. Does Blender-Network-Render-Additions allows share render of one image with some PCs in m network or not? 

My tests says it sends image only to one Slave to render. I didn't find a way to make it share parts of image among 2 or more slaves.

But it still a grate project for animation!

**Comment by kroko** [comment]
No, it does not allow rendering one image, divided in regions/tiles, across multiple slaves.

IMHO it could be done.

This code might come handy
https://wiki.blender.org/index.php/Extensions:2.6/Py/Scripts/Render/Render_Regions

As well as built in border render fcn, that could somehow be abused
https://blender.stackexchange.com/questions/2014/render-borders-related-check-boxes-how-do-they-work/2023#2023
